_data/simple-icons.json

```diff
@@ -8981,6 +8981,19 @@
             "hex": "10B146",
             "source": "https://lmms.io/branding"
         },
+        {
+            "title": "Lobsters",
+            "aliases": {
+                "aka": [
+                    "Lobste.rs"
+                ]
+            },
```

Comment:
I don't think this is necessary, they don't use this name anywhere else - it just seems to be their domain.
```suggestion
```